### ISSUE TITLE
training loss is bigger and biggere

### ISSUE THREAD

**Issue by HandsLing** [mozilla/DeepSpeech#2892]
For support and discussions, please use our [Discourse forums](https://discourse.mozilla.org/c/deep-speech).

If you've found a bug, or have a feature request, then please create an issue with the following information:

- **Have I written custom code (as opposed to running examples on an unmodified clone of the repository)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 18.04
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: Anaconda
- **TensorFlow version (use command below)**:  1.14.0
- **Python version**: 3.6.2
- **Bazel version (if compiling from source)**: None
- **GCC/Compiler version (if compiling from source)**: None
- **CUDA/cuDNN version**: 10.1 / 7.6.5
- **GPU model and memory**: 1080Ti / 11G
- **Exact command to reproduce**:

You can obtain the TensorFlow version with
i train deepspeech with LibriSpeech dataset, but when i start to train from zero, i noticed that the loss is from 30 to over 200 in a batch, wath should i do?

**Comment by lissyx** [comment]
This is not a bug, please use Discourse.

**Comment by HandsLing** [comment]
@lissyx But no matter what dataset I use, the loss is getting bigger，

**Comment by lissyx** [comment]
> @lissyx But no matter what dataset I use, the loss is getting bigger，

So what? You expect help without sharing any details on how you train and not following basic instructions such as using discourse for support?
